client.go

```diff
@@ -4,40 +4,364 @@ import (
 	"context"
 	"fmt"
 	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+	"github.com/hazelcast/hazelcast-go-client/sql"
 	"github.com/hazelcast/hazelcast-go-client/types"
 	"log"
 	"math/rand"
-	"os"
 	"time"
 )
 
-/**
- *
- * This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
- * After successful connection, it puts random entries into the map.
- *
- * See: https://docs.hazelcast.cloud/docs/go-client
- *
- */
+// This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
+// After successful connection, it runs the uncommented examples.
+//
+// See: https://docs.hazelcast.com/cloud/go-client
 func main() {
-	_ = os.Setenv("HZ_CLOUD_COORDINATOR_BASE_URL", "YOUR_DISCOVERY_URL")
 	ctx := context.Background()
 	config := hazelcast.NewConfig()
 	config.Cluster.Name = "YOUR_CLUSTER_NAME"
-	config.Cluster.Network.SSL.Enabled = false
 	config.Cluster.Cloud.Enabled = true
 	config.Cluster.Cloud.Token = "YOUR_CLUSTER_DISCOVERY_TOKEN"
 	config.Stats.Enabled = true
 	config.Stats.Period = types.Duration(time.Second)
-
 	client, err := hazelcast.StartNewClientWithConfig(ctx, config)
 	if err != nil {
 		panic(err)
 	}
 	defer client.Shutdown(ctx)
-	log.Println("Connection Successful!")
-	log.Println("Now, `map` will be filled with random entries.")
+	fmt.Println("Connection Successful!")
+
+	// Uncomment the example you want to run.
+	mapExample(client)
+	// sqlExample(client)
+	// jsonSerializationExample(client)
+	// nonStopMapExample(client)
+}
+
+func city(country string, name string, population int64) serialization.JSON {
+	return serialization.JSON(fmt.Sprintf(`{"country":"%s", "city": "%s", "population":"%d"}`, country, name, population))
+}
+
+func country(country string, name string) serialization.JSON {
```

Comment:
Changing the country parameter name to "isoCode" may be more understandable. Wdyt?